qa/resources/template_predefined_genesis.conf

```diff
@@ -0,0 +1,40 @@
+scorex {
+  dataDir = "%(DIRECTORY)s/sc_node%(NODE_NUMBER)s/blockchain"
+  logDir = "%(DIRECTORY)s/sc_node%(NODE_NUMBER)s/log"
+
+  restApi {
+    bindAddress = "%(API_ADDRESS)s:%(API_PORT)s"
+    api-key-hash = ""
+    timeout = 5s
+  }
+
+  network {
+    nodeName = "node%(NODE_NUMBER)s"
+    bindAddress = "%(API_ADDRESS)s:%(BIND_PORT)s"
+    knownPeers = []
+    agentName = "2-Hop"
+  }
+
+  websocket {
+    address = "ws://localhost:8888"
+    connectionTimeout = 100 milliseconds
+    reconnectionDelay = 1 seconds
+    reconnectionMaxAttempts = 1
+  }
+
+  wallet {
+    seed = "%(WALLET_SEED)s"
+    genesisSecrets = ["6882a61d8a23a9582c7c7e659466524880953fa25d983f29a8e3aa745ee6de5c0c97174767fd137f1cf2e37f2e48198a11a3de60c4a060211040d7159b769266",
+    "905e2e581615ba0eff2bcd9fb666b4f6f6ed99ddd05208ae7918a25dc6ea6179c958724e7f4c44fd196d27f3384d2992a9c42485888862a20dcec670f3c08a4e",
+    "80b9a06608fa5dbd11fb72d28b9df49f6ac69f0e951ca1d9e67abd404559606be9b36fb5ae7e74cc50603b161a5c31d26035f6a59e602294d9900740d6c4007f"]
```

Comment:
Maybe this part should be as previous depend on Node name/number. In other case, if we want to do standalone SC test with several nodes, all of them will control the same balance.

Reply:
Done